### PR TITLE
Issue511 j modelica compliance

### PR DESCRIPTION
**Pull request by Mathadon**
Some changes to IDEAS that used to generate errors in JModelica. This is for #511

### CODE REVIEW

**Comment by rubenbaetens** [comment]
There is a conflict due to new changes in the develop branch.

**Comment by Mathadon** [comment]
will update this after the two other pull requests are merged, since otherwise I'd have to rerun unit tests again anyway.

File: IDEAS/Buildings/Components/Interfaces/ThermalBridge.mo [hunk 1]
```diff
@@ -1,7 +1,7 @@
 within IDEAS.Buildings.Components.Interfaces;
 record ThermalBridge "Record data for thermal bridges"
 
-  parameter Real G = psi*lin "Effective thermal loss";
+  parameter Modelica.SIunits.ThermalConductance G "Effective thermal loss";
```
Comment (damienpicard) [inline IDEAS/Buildings/Components/Interfaces/ThermalBridge.mo]: Why not keep the default value?

**Comment by damienpicard** [comment]
commit cb187c1 are all white space. This should not be merged.

**Comment by damienpicard** [comment]
Apart from my comments + the conflicts, this can be merged.

File: IDEAS/Buildings/Components/Interfaces/ThermalBridge.mo (hunk 1 above)
Comment (Mathadon) [inline IDEAS/Buildings/Components/Interfaces/ThermalBridge.mo]: because psi and lin are not declared in that record

**Comment by Mathadon** [comment]
cb187c1 fixes the indentation, so it can be merged imo

**Pull request merged**